Get_a_name/99_bottles/99_bottles.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+for ((i=100
+	..; i>=1; i--));do
```

Comment:
@nsuitter Sorry Nathan! It should be for ((i=99; i>=1; i--));do.  Should we repush our assignment?